packages/tabs/test/tabs-overflow.test.ts

```diff
@@ -157,4 +189,21 @@ describe('TabsOverflow', () => {
         const slotContent = slot?.assignedElements() || '';
         expect(slotContent[0].toString()).to.not.contains('Tabs');
     });
+
+    it('should automatically bring the selected tab into view', async () => {
+        const spy = stub(HTMLElement.prototype, 'scrollIntoView');
```

Comment:
Do we really want to stub a method, which is an implementation detail and can change, or would it be better to use `el.getBoundingClientRect()` to see that the child exists within the bounds of the parent which would be the experience we're specifically looking to deliver to the end user?

Reply:
Nope, will try to calculate the `boundaries` instead!